components/eam/src/chemistry/utils/modal_aero_calcsize.F90

```diff
@@ -75,11 +75,11 @@ module modal_aero_calcsize
 integer, parameter :: npair_csizxf = N_DIAG           !total number of possible diagnostic calls
 
 logical :: do_adjust_allowed                          !flag to turn on/off  aerosol size adjustment process
-#if ( defined MODAL_AERO_5MODE )
-logical :: modal_strat_sulfate_aod = .true.           !kzm !flag to turn on/off stratospheric H2SO4 AOD treatment
-#else
-logical :: modal_strat_sulfate_aod = .false.          !no this setting in MAM4
-#endif
+!#if ( defined MODAL_AERO_5MODE )
+!logical :: modal_strat_sulfate_aod = .false.           !kzm !flag to turn on/off stratospheric H2SO4 AOD treatment
```

Comment:
Should the logical namelist variables include "_treatment" in the name? or this is a local variable that is different from the one defined for the namelist?

Reply:
I think that file, pp_chemUCI_linozv3_mam5_resus_mom_soag_tag_cnst_no3_oh2.in, is for chemUCI-mam5 only. VBS is not included.

Reply:
@hwangacme and @mingxuanwupnnl This chemistry file is for MAM5 only. This is updated file let user can do MAM5-UCI simulations. It was missed in this repository.